Jenkinsfile

```diff
@@ -44,22 +44,21 @@ pipeline {
         stage('Deliver Docker images') {
           when {
             anyOf {
-              branch 'master'
+              branch 'postgresql'
               buildingTag()
             }
           }
           steps {
             script {
-              env.DOCKER_TAG = 'branch-master'
+              env.DOCKER_TAG = 'branch-postgresql'
```

Comment:
```suggestion
              env.DOCKER_TAG = 'postgresql-experimental'
```

Reply:
Below we have: `postgres-${env.DOCKER_TAG}`

-> final CI tag:  `postgres-postgresql-experimental`.

Maybe we should remove the `postgres-` prefix below.